crates/avalanche-types/src/subnet/rpc/vm/server.rs

```diff
@@ -977,15 +977,34 @@ where
         _req: Request<Empty>,
     ) -> std::result::Result<Response<vm::VerifyHeightIndexResponse>, tonic::Status> {
         log::debug!("verify_height_index called");
-        Err(tonic::Status::unimplemented("verify_height_index"))
+
+        let inner_vm = self.vm.read().await;
+
+        inner_vm
+            .verify_height_index()
+            .await
+            .map_err(|e| tonic::Status::unknown(e.to_string()))?;
+
+        Ok(Response::new(vm::VerifyHeightIndexResponse { err: 0 }))
```

Comment:
your going to want to handle `err` basically there are a few errors which we do not return[1], [2].

[1] https://github.com/ava-labs/avalanche-rs/blob/de6fba582a86d2b2e0f2a8df1f3d2685abd3199c/crates/avalanche-types/src/subnet/rpc/vm/server.rs#L435C26-L435C45

[2] https://github.com/ava-labs/avalanchego/blob/b820b46ee8d51ab1ae05da3294ff941e6f901137/database/rpcdb/errors.go#L23